index.html

```diff
@@ -40,15 +41,15 @@
 
         <div id="mobile-menu-container">
           <span class="close-icon" onclick="menuFunction()"
-            ><img src="assets/images/close-icon.svg" alt=""
+            ><img src="image/close-icon.svg" alt="closeicon"
           /></span>
           <ul id="mylink">
             <li>
               <a
                 href="javascript:void(0);"
                 onclick="menuOptions('#my-recent-work')"
-                >Projects</a
-              >
+                >Portfolio</a>
+              
             </li>
             <li>
               <a href="javascript:void(0);" onclick="menuOptions('#about')"
```

Comment:
- [ ] Well done :+1: your project all functionalities are working but there are some design issues, please stick to the Figma design check below the comparison between your design and Figma design :point_down: 

**Your Design**
![Screenshot from 2023-03-13 13-55-09](https://user-images.githubusercontent.com/45226848/224656914-9206434d-827c-4edb-8bfd-426ac64c98a3.png)

**Figma Design**
![Screenshot from 2023-03-13 14-06-56](https://user-images.githubusercontent.com/45226848/224656981-82f43238-85be-4d71-9bd9-d9eca73346c4.png)

- [ ] If you check your `font style` and Figma design `font style` it does not match so kindly fix this issue. :slightly_smiling_face: 
- [ ] If you check the spacing between the text it does not match to the Figma design so please fix this issue. :+1: